plugins/CuraEngineBackend/StartSliceJob.py

```diff
@@ -148,7 +148,12 @@ def run(self):
 
                     obj = group_message.addRepeatedMessage("objects")
                     obj.id = id(object)
-                    verts = numpy.array(mesh_data.getVertices())
+                    verts = mesh_data.getVertices()
+                    indices = mesh_data.getIndices()
+                    if not indices is None:
```

Comment:
Codestyle. SHould be 'if indicides is not None' (Makes it easier to read what is going on)